components/GroupFilterModal.js

```diff
@@ -77,16 +77,6 @@ export default ({
       propagateSwipe={true}
       style={styles.modal}
     >
-      <AntDesign.Button
```

Comment:
you don't like the close button?? 😅

Reply:
@dereckquock it was beautiful, but we can use the extra screen real estate--especially for older iOS devices and smaller android devices--and it wasn't necessary.